packages/table/src/Table.tsx

```diff
@@ -43,8 +43,10 @@ export default function Table<Shape>({
         className={cx(tableStyles, className)}
         {...rest}
       >
-        <TableHead columns={columns} />
```

Comment:
Should we move `<TableProvider>` in here as well? As it has no impact to `<table`?